exercises/flower-field/canonical-data.json

```diff
@@ -0,0 +1,119 @@
+{
+  "exercise": "flower-field",
+  "comments": [
+    "The expected outputs are represented as arrays of strings to",
+    "improve readability in this JSON file.",
+    "Your track may choose whether to present the input as a single",
+    "string (concatenating all the lines) or as the list."
+  ],
+  "cases": [
+    {
+      "uuid": "237ff487-467a-47e1-9b01-8a891844f86c",
+      "description": "no rows",
+      "property": "annotate",
+      "input": {
+        "garden": []
+      },
+      "expected": []
+    },
+    {
+      "uuid": "4b4134ec-e20f-439c-a295-664c38950ba1",
+      "description": "no columns",
+      "property": "annotate",
+      "input": {
+        "garden": [""]
+      },
+      "expected": [""]
+    },
+    {
+      "uuid": "d774d054-bbad-4867-88ae-069cbd1c4f92",
+      "description": "no flowers",
+      "property": "annotate",
+      "input": {
+        "garden": ["   ", "   ", "   "]
```

Comment:
While the linting change can make some sense, having it as it was can make it easier to see that the space lengths are identical (apparently) from the alignment of the quotes.  If they are not spaces and some other unprinted white space, then that can be harder to immediately see, but when thinking of four legs, hooves and manes, we think "horse" rather than "zebra" (unless of course, you are more accustomed to seeing zebras).

So I understand the linting change, but miss having the nice easily seen "all the same" from having the individual lines.  Plus if one element is changed, it is also cleaner in diffs.

So there are benefits to having these the other way.

Reply:
Adjusted the ruby linting rule -- hope that's okay.
Multi-line array is back to being on multiple lines now.